docs/guides/advanced-cluster-new-sharding-schema.md

```diff
@@ -360,7 +360,14 @@ resource "mongodbatlas_advanced_cluster" "test" {
 }
 ```
 
--> **NOTE:** For any cluster leveraging the new sharding configurations and defining independently scaled shards, users should also update corresponding `mongodbatlas_cloud_backup_schedule` resource & data sources. This involves updating any existing Terraform configurations of the resource to use `copy_settings.#.zone_id` instead of `copy_settings.#.replication_spec_id`. This is needed as `mongodbatlas_advanced_cluster` resource and data source will no longer have `replication_specs.#.id` present when shards are scaled independently. To learn more, review the [1.18.0 Migration Guide](1.18.0-upgrade-guide.md#transition-cloud-backup-schedules-for-clusters-to-use-zones).
+### Resources and Data Sources Impacted by Independent Shard Scaling
+
+Name | Changes | Transition Guide
```

Comment:
This table and section is great 👏  One thing to note, under the [Use Auto Scaling Per Shard](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/guides/advanced-cluster-new-sharding-schema#use-auto-scaling-per-shard) there is also a note on impact of clusters transitioning to asymmetric shards, could we adjust that note as well and have them both reference this common section? Thinking we can move this towards the bottom.